packages/web/src/ui/tools/checkPlatformVersion.ts

```diff
@@ -0,0 +1,17 @@
+export const checkPlatformVersion = (
+  requiredV: string,
+  currentV: string | undefined
+) => {
+  if (!currentV || currentV === '??' || currentV === 'local') {
```

Comment:
You should rather check whether the version is semver and if not, let it pass. Sometimes, we also release other versions for testing purposes on testing instance.